src/Microsoft.Crank.PullRequestBot/Microsoft.Crank.PullRequestBot.csproj

```diff
@@ -28,17 +28,12 @@
     <PackageReference Include="Octokit" Version="0.48.0" />
     <PackageReference Include="Newtonsoft.Json" Version="$(JsonNetVersion)" />
     <PackageReference Include="YamlDotNet" Version="8.1.2" />
-    <PackageReference Include="Manatee.Json" Version="13.0.5" />
     <PackageReference Include="Fluid.Core" Version="2.2.6" />
 
     <!-- Force version for Component Governance compliance -->
-    <PackageReference Include="System.Net.Http" Version="4.3.4">
+    <PackageReference Include="System.Text.Encodings.Web" Version="6.0.0">
```

Comment:
This might not even be necessary anymore. I assume STEW was referenced directly by another lib (Manatee?) and then I had to force it to a more recent version. If you could try to build the solution locally without the reference, and it restores something >= 4.3.4 then we can remove this package ref.

Reply:
After removing this package ref, it can be successfully built, and all tests have passed. 

![image](https://user-images.githubusercontent.com/8394988/222012365-60e4168b-5086-4d2e-a432-e4a591f34b0f.png)

Reply:
And do you see what version it restored? I will also follow the version trail to see which one it would use.

Reply:
**System.Net.Http** was removed.

**System.Text.Encodings.Web** is 6.0.0

Reply:
Thanks, should be good then. Still waiting for the packages to be published.